package.json

```diff
@@ -46,8 +49,34 @@
     "@julusian/jpeg-turbo": "^0.5.4",
     "@slack/client": "^3.5.4",
     "android-device-list": "^1.2.1",
+    "angular": "1.5.11",
+    "angular-animate": "1.5.11",
+    "angular-borderlayout": "https://github.com/filearts/angular-borderlayout#7c9716aebd9260763f798561ca49d6fbfd4a5c67",
+    "angular-cookies": "1.5.11",
+    "angular-elastic": "2.5.1",
+    "angular-gettext": "2.2.1",
+    "angular-growl-v2": "https://github.com/JanStevens/angular-growl-2#v0.7.9",
+    "angular-hotkeys": "1.6.0",
+    "angular-ladda": "0.3.3",
+    "angular-route": "1.5.11",
+    "angular-touch": "1.5.11",
+    "angular-ui-bootstrap": "1.1.2",
+    "angular-xeditable": "0.1.12",
     "aws-sdk": "^2.4.13",
+    "packery": "1.4.3",
+    "components-font-awesome": "4.7.0",
+    "draggabilly": "1.2.4",
+    "epoch-charting": "0.8.4",
+    "underscore.string": "3.2.3",
+    "d3": "3.5.17",
+    "ng-context-menu": "https://github.com/swimlane/ng-context-menu#93d825d981dc6f49bcaa861ef5a87fa9d1923f34",
+    "ng-epoch": "file:ng-epoch",
+    "ng-file-upload": "2.2.2",
+    "ng-table": "1.0.0",
+    "oboe": "2.1.5",
+    "font-lato-2-subset": "file:font-lato-2-subset",
```

Comment:
You should publish your `font-lato-2-subset` package to `npmjs`, and not reference it locally at code root or at least put it properly in `./res` folder!

Reply:
I think this is not good idea since this package is quite old

Reply:
i will put it to .res

Reply:
fixed